VMware.CloudFoundation.CertificateManagement.psm1

```diff
@@ -1319,6 +1319,415 @@ Function Install-EsxiCertificate {
     }
 }
 
+Function Set-SddcCertificateAuthority {
+    <#
+        .SYNOPSIS
+        Configure Microsoft Certificate Authroity in SDDC Manager
+
+        .DESCRIPTION
+        The Set-SddcCertificateAuthority will configure Microsoft Certificate Authority as 
+        SDDC Manager's Certificate Authority.
+
+        .EXAMPLE
+        Set-SddcCertificateAuthority -server sfo-vcf01.sfo.rainpole.io -user administrator@vsphere.local -pass VMw@re1! -vcfCertCaFqdn 
+        rpl-ad01.rainpole.io -vcfCertCaUsername svc-vcf-ca -vcfCertCaPassword VMw@re1! -vcfCertCaTemplate VMware
+        This example will configure Microsoft Certificate Authority rpl-ad01.rainpole.io in SDDC Manger.
+
+        .PARAMETER server
+        The fully qualified domain name of the SDDC Manager instance.
+
+        .PARAMETER user
+        The username to authenticate to the SDDC Manager instance.
+
+        .PARAMETER pass
+        The password to authenticate to the SDDC Manager instance.
+
+        .PARAMETER vcfCertCaFqdn
+        The fully qualified domain name of the Microsoft Certificate Authority.
+
+        .PARAMETER vcfCertCaUsername
+        The username to authenticate to the Microsoft Certificate Authority.
+
+        .PARAMETER vcfCertCaPassword
+        The password to authenticate to the Microsoft Certificate Authority.
+
+        .PARAMETER vcfCertCaTemplate
+        The Certificate Template Name to be used with the Microsoft Certificate Authority.
+    #>
+
+    Param (
+        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String] $server,
+        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String] $user,
+        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String] $pass,
+        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String] $vcfCertCaFqdn,
+        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String] $vcfCertCaUsername,
+        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String] $vcfCertCaPassword,
+        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String] $vcfCertCaTemplate
+    )
+
+    # Connect to VMware SDDC Manager
+    if (Test-VCFConnection -server $server) {
+        $connectionStatus = Test-NetConnection -Port 443 -ComputerName $vcfCertCaFqdn
+        if ($connectionStatus.TcpTestSucceeded -eq "True" ) {
+            if (Test-VCFAuthentication -server $server -user $user -pass $pass) {
+                $vcfVersion = Get-VCFManager | select version | Select-String -Pattern '\d+\.\d+' -AllMatches | ForEach-Object {$_.matches.groups[0].value}
+                $serverUrl = "https://$vcfCertCaFqdn/certsrv"
+                Try
+                {
+                    Write-Output "Starting the Process of Configuring a Microsoft Certificate Authority in SDDC Manager"
```

Comment:
```suggestion
                    Write-Output "Starting configuration of a Microsoft Certificate Authority in SDDC Manager..."
```

Reply:
addressed in (131acd4bec5c68a803eb55ef8c6b8d2638b81d17)